dream_layer_backend/extras.py

```
@@ -313,6 +314,88 @@ def upscale_image():
+    """Generate a batch report ZIP file containing CSV, JSON, images, and README"""
+    if request.method == 'OPTIONS':
+        # Handle preflight request
+        response = jsonify({'status': 'ok'})
+        response.headers.add('Access-Control-Allow-Origin', 'http://localhost:8080')
+        response.headers.add('Access-Control-Allow-Headers', 'Content-Type')
+        response.headers.add('Access-Control-Allow-Methods', 'POST, OPTIONS')
+        return response
+
+    try:
+        data = request.json
+        print(f"📊 Batch report generation request received")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)